### DISCUSSION TITLE
OsmAnd Live Tracking web address format for ESP32

### DISCUSSION THREAD

**Discussion by zero48** [osmandapp/OsmAnd#15425] [Ideas]
Hello, 
Wondering if anybody implemented OsmAnd Live Tracking web address format On an ESP32 Microcontroller ?
I adapted my GPS platform (non Traccar) to receive data from Traccar Apps which uses OsmAnd protocol and works great . My intention is to use the ESP32 along with My GPS system IOT capabilities/alarm/triggering rules to measure some GPIO .
http://demo.traccar.org:5055/?id=123456&lat={0}&lon={1}&timestamp={2}&hdop={3}&altitude={4}&speed={5}
SO i may add almost anything ... However at ESP32 side i was wondering how to do it ... I use Platformio.
If Somebody interested in join efforts I can Open an GPS account for it

**Comment by gh3597** [comment]
Did you ever get anywhere with this idea? I'm just starting this now as I can't find a tracking device that works how I want. Already have a mobile 4G WiFi router in a vehicle so just need GPS data to be sent to a Traccar server using POST/GET. Found a spare NodeMCU so will start with that and see how we get on....
